ipc/provider/src/checkpoint.rs

```diff
@@ -105,6 +138,11 @@ impl<T: BottomUpCheckpointRelayer + Send + Sync + 'static> BottomUpCheckpointMan
     pub async fn run(self, submitter: Address, submission_interval: Duration) {
         log::info!("launching {self} for {submitter}");
 
+        const PROMETHEUS_EXPORTER_URL: &str = "0.0.0.0:9185";
```

Comment:
* Note that this is not a URL, it's a bind address.
* Let's make this a CLI argument to the relayer command (`--prometheus-bind`) with a default value.

Reply:
Fixed as you mentioned.